libdebug/libdebug.py

```diff
@@ -382,18 +472,114 @@ def watchpoint(
             callback=callback,
         )
 
+    def _enable_pretty_print(
+        self,
+    ) -> SyscallHook:
+        """Hooks a syscall in the target process to pretty prints its arguments and return value."""
+        self._ensure_process_stopped()
+
+        syscall_numbers = get_all_syscall_numbers()
+
+        for syscall_number in syscall_numbers:
+            # Check if the syscall is already hooked (by the user or by the pretty print hook)
+            if syscall_number in self.context.syscall_hooks:
+                hook = self.context.syscall_hooks[syscall_number]
+                if syscall_number not in (
+                    self.context._syscalls_to_not_pprint or []
```

Comment:
Nope, it is needed. _syscalls_to_not_pprint, when not used, is None, not an empty ([]) list. 
This is because _syscalls_to_pprint, when is the empty list, has a very different meaning to None. For coherence, also _syscalls_to_not_pprint follows this convention.
The boolean condition avoid to check if the value is not None in a separate branch

Reply:
Yeah, I figured that they were needed, they just don't look elegant